project_name/utils/blocks.py

```diff
@@ -125,7 +125,7 @@ class Meta:
 
 class FeaturedArticleBlock(blocks.StructBlock):
```

Comment:
This block isn't used anywhere, it's imported in `news/models.py` but not used. I'm not sure how the project template is supposed to be used. Do we only provide the block classes, and users get to choose which ones to include in the `StreamField` definition? Most of the blocks don't have meaningful templates too (#38).

Reply:
I want to create a template for this block because I feel like it's a valuable one to show people how to use and how to incorporate into their projects.

When I went to shoot a tutorial on the starter kit, I was thinking I would have to add the extra blocks myself and surprised to find a bunch of extra blocks without templates or malfunctioning templates were included.

I think this one is worth fixing up. I've gone ahead and provided fixes for several others listed in #38 as well.